PatientMonitoringSystem/PatientMonitoringSystem/Controllers/BedsideSystemController.cs

```diff
@@ -17,11 +18,17 @@ public class BedsideSystemController
 
 		private readonly int maxModulesPerBedsideSystem;
 
+		private Timer poller;
+
 		public BedsideSystemController(BedsideSystemView bedsideSystemView, Guid bedsideSystemId)
 		{
 			this.bedsideSystemView = bedsideSystemView;
 			bedsideSystem = Program.BedsideSystems.Single(bs => bs.BedsideSystemId == bedsideSystemId);
 			maxModulesPerBedsideSystem = int.Parse(ConfigurationManager.AppSettings["MaxModulesPerBedsideSystem"]);
+			poller = new Timer();
+			poller.Elapsed += new ElapsedEventHandler(UpdateCurrentReading);
```

Comment:
@harrytpstudent Wasn't the poller just meant to retrieve the current reading, and the modules were meant to update themselves?

Reply:
The poller is retrieving the current reading.  What do you mean by the modules updating themselves?